codes/c/chapter_array_and_linkedlist/my_list.c

```diff
@@ -0,0 +1,150 @@
+/**
+ * File: list.c
+ * Created Time: 2022-01-12
+ * Author: Zero (glj0@outlook.com)
+ */
+
+#include "../include/include.h"
+
+
+// 用数组实现 list
+struct mylist {
+    int* nums;       // 数组（存储列表元素）
+    int capacity;    // 列表容量
+    int size;        // 列表大小
+    int extendRatio; // 列表每次扩容的倍数
+};
+
+typedef struct mylist MyList;
+/* 前置声明 */
+void extendCapacity(MyList* l);
```

Comment:
Could we use `list` instead of `l`? I will help improve consistency.

Reply:
> 

hello, i have finished processing the comment.